### PR TITLE
MMrazor Distillation for Object Detection, Segmentation and Pose Estimation

### PR DESCRIPTION
**Pull request by Akash-guna**
Adds MMRazor Distillation for Object Detection, Segmentation and Pose Estimation

### CODE REVIEW

**Review by shwu-nyunai — COMMENTED** [review]
added a few comments. if no changes are required, resolve.

File: distill/mmrazordistill/main.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        self.logger.info(f"Teacher Pth:{self.teacher_pth}")
+        if self.wandb:
+            wandb.init(project="Kompress MMRazor", name=str(self.job_id))
+            wandb.config.update(self.kwargs)
+
+    def write_config(self):
+        pass
+
+    def customize_config(self):
+        pass
```
[the remark this answers is not in the document]
Comment (Akash-guna) [reply distill/mmrazordistill/main.py]: added that

File: distill/mmrazordistill/utils.py [hunk 2]
```diff
@@ -0,0 +1,947 @@
+def find_folder(cfg):
+    model = cfg.split(".")[0]
+    model = model.split("_")[0]
+    if "faster-rcnn" in model:
```
[the remark this answers is not in the document]
Comment (Akash-guna) [reply distill/mmrazordistill/utils.py]: yeah checked now, could be replaced was added by intern

**Review by shwu-nyunai — APPROVED** [review]
💫 awsm. thanks @Akash-guna

**Pull request merged**